cypress/e2e/surveys.cy.js

```diff
@@ -0,0 +1,122 @@
+/// <reference types="cypress" />
+
+function onPageLoad() {
+    cy.posthogInit(given.options)
+    cy.wait('@decide')
+    cy.wait('@surveys')
+}
+
+describe('Surveys', () => {
+    given('options', () => ({}))
+    beforeEach(() => {
+        cy.intercept('POST', '**/decide/*', {
+            config: { enable_collect_everything: false },
+            editorParams: {},
+            featureFlags: ['session-recording-player'],
+            surveys: true,
+            isAuthenticated: false,
+        }).as('decide')
+    })
+    it('shows and submits a basic survey', () => {
+        cy.intercept('GET', '**/surveys/*', {
+            surveys: [
+                {
+                    id: '123',
+                    name: 'Test survey',
+                    active: true,
+                    type: 'popover',
+                    start_date: '2021-01-01T00:00:00Z',
+                    questions: [{ type: 'open', question: 'What is your role?' }],
+                },
+            ],
+        }).as('surveys')
+        cy.visit('./playground/cypress')
+        onPageLoad()
+        cy.wait(500)
+        const survey = cy.get('.PostHogSurvey123').shadow()
+        survey.find('.survey-123-form').should('be.visible')
+        survey.find('.question-textarea-wrapper').type('product engineer')
```

Comment:
can you also assert the question & description is correct?

And cover all types of surveys?

Reply:
and also intercept the `/e/` or `/batch` endpoint to confirm that the right events are sent [dismiss / sent with the right data]

Reply:
The types of surveys should be covered by the multiple question survey test~

Reply:
Should be a good base to build off of for now

Reply:
sweet 👍